web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -6,4 +6,13 @@
     <session-config>
         <session-timeout>5</session-timeout>
     </session-config>
+
+    <error-page>
+        <exception-type>java.lang.Throwable</exception-type>
+        <location>/error</location>
+    </error-page>
+    <error-page>
+        <error-code>404</error-code>
```

Comment:
In Spring Boot — yes. In case of non-Boot configuration those errors get intercepted by Tomcat. As a result Tomcat's standard pages are served to the user, not the application's ones.